test/test_write_cache.c

```diff
@@ -51,6 +54,7 @@ static const struct fuse_opt option_spec[] = {
     FUSE_OPT_END
 };
 static int got_write;
+static atomic_int write_cnt;
```

Comment:
Is this used anywhere?

Reply:
It has to - is a core requirement of this patch

```
assert(write_cnt < WRITE_SYSCALLS);
```

Reply:
Complete

```
    if (options.writeback)
        assert(write_cnt < WRITE_SYSCALLS);
    else
        assert(write_cnt == WRITE_SYSCALLS);
``

Reply:
Ah yes, that makes much more sense, I'm not sure how I missed that.